### ISSUE TITLE
Indentation problem in defgeneric form

### ISSUE THREAD

**Issue by nneuss** [slime/slime#408]
I observe the following indentation using SLIME 2.19 from Quicklisp, i.e. the first line of the method body is indented more than the rest:
(defgeneric foo (x)
  (:method (x)
      (print x)
    (* x x)))

**Comment by luismbo** [comment]
Can you reproduce this with a minimal configuration and share it here?

**Comment by nneuss** [comment]
This is a good point.
A minimal feature set of SLIME features reproducing the problem is
 (slime-setup '(slime-cl-indent))
Without slime-cl-indent, indentation seems to work fine.  I would be satisfied with dropping slime-cl-indent, but if slime-cl-indent should work as well and you are interested, I can investigate further.

**Comment by luismbo** [comment]
Hmm, yes, I'm interested in knowing why slime-cl-indent isn't working for you. (Even though you're meant to load it via slime-indentation, though.) 

I can't reproduce the issue though. The devil must be in some subtle detail. Can you came up with a more precise recipe?

**Comment by luismbo** [comment]
Marking _irreproducible_ until we have further news.

**Comment by nneuss** [comment]
Luís Oliveira <notifications@github.com> writes:

> Hmm, yes, I'm interested in knowing why slime-cl-indent isn't working
> for you. (Even though you're meant to load it via slime-indentation,
> though.)
>
> I can't reproduce the issue though. The devil must be in some subtle
> detail. Can you came up with a more precise recipe?

I have now looked again and tested without initialization files.  It
turned out that it was completely my fault.  I had another defmethod
indentation rule from before-SLIME times which was loaded via "(require
'femlisp)" (purpose of this file is/was: color and indent some Femlisp
macros) and which did not work anymore (probably due to some change in
Emacs).  I dropped this defmethod-rule, and everything works fine.

So closing this bug report was the right thing to do.

Thank you,
Nicolas
